yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
+-- > get HomeR
+--
+-- > followRedirect
+followRedirect :: Yesod site
+               =>  YesodExample site ()
+followRedirect = do
+  mr <- getResponse
+  case mr of
+   Nothing ->  failure "no response, so no redirect to follow"
+   Just r -> do
+     if not ((H.statusCode $ simpleStatus r) `elem` [301,303])
```

Comment:
I'm not super familiar this, but should other status codes be considered? https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection

It seems like 308 is a valid HTTP code for a redirect?